aws-backend/amplify/backend/api/fellowbookapi/api-params.json

```diff
@@ -57,6 +57,13 @@
             "privacy": {
                 "open": true
             }
+        },
+        {
+            "name": "/discord-message",
+            "lambdaFunction": "discordwebhookendpoint",
```

Comment:
Probably better to have a more specific name, because we'll have: (1) webhook for GitHub starring event, (2) webhook for Linkedin endorsement event, (3) webhook for Most Valuable Fellow.

Reply:
Updated to require a new `message_type` parameter, so we use the same endpoint for all the messages!